File: benchmarks/bench_12.go
package main

import (
	"fmt"
	"time"
)

func type_cast() int64 {
    c := int64(0)
    i := int64(0)
    val := 1.5
    for i < 2000000 {
        c += int64(val)
        i += 1
    }
    return c
}

func main() {
    t1 := time.Now()
    res := type_cast()
    t2 := time.Now()
    fmt.Println(res)
    fmt.Printf("%.6f\n", t2.Sub(t1).Seconds())
}

File: benchmarks/bench_12.py
import time
import sys
sys.setrecursionlimit(200000)

def type_cast():
    c = 0
    i = 0
    while i < 2000000:
        c += int(1.5)
        i += 1
    return c


t1 = time.perf_counter()
res = type_cast()
t2 = time.perf_counter()
print(res)
print(f"{t2 - t1:.6f}")
